modules/alloy/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
+### Added
+- Added the `onToggled` callback for `SplitFloatingToolbar` and `SplitSlidingToolbar` components. #TINY-9271
```

Comment:
I don't think is accurate any more. It seems more like the FloatingToolbar added onOpened and onClosed, and the FloatingToolbarButton added an onToggled

Reply:
Updated the changelog [here](https://github.com/tinymce/tinymce/pull/8238/commits/521c870d2e05af4fae59b2dd1a3c66cd80715894)